rules/no-skip-assert.js

```diff
@@ -11,7 +11,7 @@ const create = context => {
 			ava.isInTestFile,
 			ava.isInTestNode
 		])(node => {
-			if (node.property.name === 'skip' && util.nameOfRootObject(node) === 't') {
+			if (node.property.name === 'skip' && node.object.name === 't') {
```

Comment:
What if it's `t.true.skip()`?

Reply:
`skip` can be before or after assertion (`t.is.skip` or `t.skip.is`) so I fixed it but `end`, `true` and `false` should be considered only when following `t` (`t.end` and not `t.whatever.end`).

Reply:
It can only be after.

Reply:
Ok, my bad. Unit tests are broken then.